tests/bluetooth/host/common/defaults_cmake.txt

```diff
@@ -0,0 +1,55 @@
+#
+# Common include directories and source files for bluetooth unit tests
+#
+
+include_directories(
+  ${CMAKE_CURRENT_SOURCE_DIR}/
+  ${CMAKE_CURRENT_SOURCE_DIR}/src
+  ${ZEPHYR_BASE}/tests/bluetooth/host/mock_host/include
+  ${ZEPHYR_BASE}/tests/bluetooth/host/common/include
+  ${ZEPHYR_BASE}/include/zephyr/net
+  ${ZEPHYR_BASE}/include/zephyr/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth
+  ${ZEPHYR_BASE}/subsys/bluetooth/host
+  ${ZEPHYR_BASE}/subsys/testsuite/ztest/include/zephyr
+)
+
+FILE(GLOB host_sources
+  ${ZEPHYR_BASE}/subsys/bluetooth/host/buf.c
+)
```

Comment:
This does not look global to `test/bluetooth/host`. Please move this to folder `buf`, or to each test.